ui/src/style/components/annotation-editor.scss

```diff
@@ -0,0 +1,82 @@
+.annotation-editor .overlay-heading {
```

Comment:
hmm maybe it could accept a custom class, so that it's children can be styled without using the `overlay-heading` class.

or maybe this is a reasonable default style.